src/components/AnimatedStep.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import * as Animatable from 'react-native-animatable';
+
+const propTypes = {
+    /** Children to wrap in AnimatedStep. */
+    children: PropTypes.node.isRequired,
+
+    /** Styles to be assigned to Container */
+    style: PropTypes.arrayOf(PropTypes.object),
+
+    /** Whether we're animating the step in or out */
+    direction: PropTypes.string,
```

Comment:
Given that it's either in or out, I think a boolean would be fine here. Then we can make 'in' the default.